docs/source/guides/serializers.rst

```diff
@@ -40,6 +40,11 @@ The state of an optimizer can also be saved by the same functions:
    Note that serialization of optimizer only saves its internal states including number of iterations, momentum vectors of MomentumSGD, etc.
    It does not save the parameters and persistent values of the target link.
    We have to explicitly save the target link with the optimizer to resume the optimization from saved states.
+   This can be done by saving the entire :class:'~training.Trainer' object, like this:
+
+.. code-block:: console
```

Comment:
`console` -> `python` ?

Reply:
This page seems to have several problems.

* All python code is using `console`
* Cross reference (to other class/modules) are all dead

Could you check preview and fix these issues?

Reply:
Done.